src/mal/SHCore.cpp

```diff
@@ -1800,14 +1800,30 @@ BUILTIN("context-var") {
 }
 
 template <SHType T> struct GetComponentType {};
-template <> struct GetComponentType<SHType::Float2> { typedef double Type; };
-template <> struct GetComponentType<SHType::Float3> { typedef float Type; };
-template <> struct GetComponentType<SHType::Float4> { typedef float Type; };
-template <> struct GetComponentType<SHType::Int2> { typedef int64_t Type; };
-template <> struct GetComponentType<SHType::Int3> { typedef int32_t Type; };
-template <> struct GetComponentType<SHType::Int4> { typedef int32_t Type; };
-template <> struct GetComponentType<SHType::Int8> { typedef int16_t Type; };
-template <> struct GetComponentType<SHType::Int16> { typedef int8_t Type; };
+template <> struct GetComponentType<SHType::Float2> {
+  typedef double Type;
+};
+template <> struct GetComponentType<SHType::Float3> {
+  typedef float Type;
+};
+template <> struct GetComponentType<SHType::Float4> {
+  typedef float Type;
+};
+template <> struct GetComponentType<SHType::Int2> {
```

Comment:
Did you change formatting settings, if so can you commit those too? 
for me it puts these on one line using the current .clang-format rules.

Reply:
I didn't change the formatting settings. I ran `cargo fmt` and `ninja format`.